2025/03/delhi-fish-market-consent-reel/writeup.md

```diff
@@ -0,0 +1,27 @@
+# Writeup
+
+So whatever happened to this girl and the camera man in this video is wrong in every way.
+
+But I heard this girl saying in the video "Sir app chhu nahin sakte" intending that they cannot touch them and come closer to them without their consent. 
+
+But the same creator who is upset and concerned when her rights are violated will later go and consume animal product for which animals are violated day in and day out 
+
+They are tide hands are being put into their rectum hands in the vagina to make them pregnant they show every sign as if they are also saying"sir aap chu nahi sakte" but there voices are unheard the signs ignored. 
+
+So if you are not a hypocrite and morally consistent you can figure that animals and humans have a right on the body which needs to be preserved and respected and everywhere and always. 
+
+At the end of the video you can also see the atrocities committed on the animals in the dairy industry, if that makes you sob, boycot all animal products it will make a difference
```

Comment:
At the end of the video you can also see the atrocities committed on the animals in the dairy industry, if that makes you sob, boycott all animal products it will make a difference.